### DISCUSSION TITLE
Exiting slideshow with CTRL+C not working

### DISCUSSION THREAD

**Discussion by devvthedev** [helgeerbe/picframe#186] [Q&A] [answered]
Hi, I've been testing out picframe on my raspberry pi 4 (2Gb) and really like it so far!

When testing the slideshow by running:

`sudo xinit /usr/bin/python3 /home/pi/picframe_data/run_start.py /home/pi/picframe_data/config/configuration.yaml`

The picture frame starts ok but hitting `CTRL C` doesn't exit the slideshow and I have to always power off my pi.

When I do try exiting with CTRL C nothing happens.. Am I missing something with the setup?

Thanks

**Comment by helgeerbe** [comment]
Hi @devvthedev do you see any output on the console, if you hit `CTRL C`?

In the meanwhile you can run `ps axu | grep picframe` to get the pid (process id of picframe). `kill pid` should terminate the process.

**Reply by devvthedev** [reply]
Apologies for the delayed response! Managed to get around this by stopping the frame via the web server after noticing the stop button which exits the photo frame 🙂 

Cannot see any console output but can try SSH while it is running on another machine 👍

Thanks

**Comment by sedwards58** [comment]
'sudo killall picframe' or 'sudo systemctl stop picframe' may be useful.

**Comment by helgeerbe** [comment, marked as answer]
CTRL+C should work with release 2022.01.20
